index.md

```diff
@@ -3,10 +3,59 @@ layout: default
 title: Duraphilms
 ---
 
-<div style="position: sticky; top: 0; z-index: 1; height: 4em; padding-top: 0.4em; margin-bottom: 1em; background-color: white; border-bottom: 1px solid #159957">
+<style>
+.left, .right{
+    position: absolute;
+    top: 8px;
+    height: 12em; 
+    width: 5rem;
+    background-image: linear-gradient(to left, rgb(255, 255, 255), rgba(255, 255, 255, 0));
+}
+.left{
+    transform: rotate(180deg);
+}
+.right{
+    right: 0;
+}
+
+#DmButton{
+    position: absolute;
+    top: 0.5rem;
+    right: 0;
+    background-color: rgb(200, 200, 200); /* Green */
+    color: #159957;
+    padding: 10px 20px;
+    border-radius: 9px;
+    text-align: center;
+    text-decoration: none;
+    border: solid #159957 2px;
+    font-size: 16px;
+    font-weight: bold;
+}
+.DmButtonDm{
+    background-color: rgb(26, 29, 42) !important;
+}
+.DarkModeBg{
+    background-color: rgb(26, 29, 42) !important;
+}
+.LRScrollDm{
+    background-image: linear-gradient(to left, rgb(26, 29, 42), rgba(255, 255, 255, 0)) !important;
+}
+@media only screen and (max-width: 850px){
+    #DmButton{
+        position: relative;
+    }
+    #StickyHeadline{
+        margin-bottom: 5em !important;
+    }
+}
+</style>
+
+<div id="StickyHeadline" style="position: sticky; top: 0; z-index: 1; height: 4em; padding-top: 0.4em; margin-bottom: 1em; background-color: white; border-bottom: 1px solid #159957">
```

Comment:
@lnjX An der Stelle könnte man eigentlich den ganzen style-tag auslagern in den obigen style block:
```css
#StickyHeadline {
    position: sticky;
    top: 0;
    z-index: 1;
    height: 4em;
    padding-top: 0.4em;
    margin-bottom: 1em;
    background-color: white;
    border-bottom: 1px solid #159957;
}
```